output.json

```diff
@@ -1,31 +1,12 @@
 {
-    "blueprints": [{
-            "invoke": "borders.pr",
-            "write_to": "Styles/BorderExtension.swift"
-        },
-        {
-            "invoke": "colors.pr",
-            "write_to": "Styles/ColorExtension.swift"
-        },
-        {
-            "invoke": "gradients.pr",
-            "write_to": "Styles/GradientExtension.swift"
-        },
+    "blueprints": [
         {
-            "invoke": "measures.pr",
-            "write_to": "Styles/AppMeasures.swift"
+            "invoke": "styleguide.pr",
+            "write_to": "Styles/StyleGuide.swift"
         },
         {
-            "invoke": "radius.pr",
-            "write_to": "Styles/RadiusExtension.swift"
-        },
-        {
-            "invoke": "shadows.pr",
-            "write_to": "Styles/ShadowExtension.swift"
-        },
-        {
-            "invoke": "typography.pr",
-            "write_to": "Styles/FontExtension.swift"
+            "invoke": "colors.pr",
+            "write_to": "Styles/Color.swift"
         }
     ]
 }
```

Comment:
Figured we don't need to have unneeded exporters for this first release. We can always reference the original Supernova exporter repo to slowly add back in the files that we'll need as we scale up to other tokens.